packages/cli/src/utils/checks.ts

```diff
@@ -193,6 +193,16 @@ class CheckBuilder {
       this.withGovernance(async (governance) => !(await governance.getHotfixRecord(hash)).executed)
     )
 
+  hotfixExecutionTimeLimitNotReached = (hash: Buffer) =>
+    this.addCheck(
+      `Hotfix 0x${hash.toString('hex')} execution time limit has not been reached`,
```

Comment:
For a message a positive form might read/comprehend easier.

like  "Hotfix *hash* is still in its execution time limit window".